account_financial_risk/views/res_partner_view.xml

```diff
@@ -7,7 +7,7 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="base.view_partner_form" />
         <field name="priority" eval="20" />
-        <field name="groups_id" eval="[(4, ref('account.group_account_manager'))]" />
+        <field name="groups_id" eval="[(4, ref('account_financial_risk.group_overpass_partner_risk_exception'))]" />
```

Comment:
But non allowed to overpass the risk should query the info (but not change it). I think it should be a new group, and stacked them as User/Manager or similar.

Reply:
Before this change only `group_account_manager` can query the info.

After this change, `group_account_manager` can continue to query the info because `group_account_manager` has implicit `group_overpass_partner_risk_exception`:
https://github.com/OCA/credit-control/blob/858d68d3f872144d0d6f644c242131bf0400390d/account_financial_risk/security/security.xml#L8-L13

In all this time, it has not been necessary for someone without group_account_manager role can query the details of the risk.
A new group with permissions to query can cause a change in behavior in a stable version, it can be proposed for a new version if it is considered necessary.

Reply:
It's not a change in behavior with proper migration scripts including account managers in such group. I think it's logical to have a risk viewer and risk manager, don't you think?

Anyways, I'm not blocking.